_inc/client/at-a-glance/scan.jsx

```diff
@@ -56,8 +56,7 @@ const DashScan = React.createClass( {
 						<DashItem
 							label={ labelName }
 							module="scan"
-							status="is-error"
-							statusText={ __( 'Threats found' ) }
+							pro={ true }
```

Comment:
Yeah, believe it or not -- so that it's actually using the ProStatus component (which it should have been anyway), and hence trackable